desktop/funnelcake138/distribution/distribution.ini

```diff
@@ -0,0 +1,10 @@
+# Partner Distribution Configuration File
+
+[Global]
+id=mozilla138
+version=1.0
+about=Funnelcake Taskbar Pinning Control
+
+[Preferences]
+app.partner.mozilla138="mozilla138"
+startup.homepage_welcome_url="https://www.mozilla.org/%LOCALE%/%APP%/%VERSION%/firstrun/?f=138"
```

Comment:
This can be removed. We've switched to about:welcome internally, so we no longer do welcome URLs in the funnelcake.

Reply:
Will do both of these.